pulpcore/app/serializers/user.py

```diff
@@ -289,6 +305,16 @@ class GroupRoleSerializer(ModelSerializer, NestedHyperlinkedModelSerializer):
         allow_null=True,
     )
 
+    description = serializers.SerializerMethodField()
+    permissions = serializers.SerializerMethodField()
+
+    def get_description(self, obj):
+        return obj.role.description
+
+    @extend_schema_field(serializers.ListField(child=serializers.CharField()))
+    def get_permissions(self, obj):
```

Comment:
Would this achieve the same result?

```suggestion
    from typing import List
    
    def get_permissions(self, obj) -> List[str]:
```

Reply:
oooh, this works. Very cool.